framework/source/class/qx/test/log/Filters.js

```diff
@@ -0,0 +1,64 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2004-2009 1&1 Internet AG, Germany, http://www.1und1.de
+
+   License:
+     LGPL: http://www.gnu.org/licenses/lgpl.html
+     EPL: http://www.eclipse.org/org/documents/epl-v10.php
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+ * Fabian Jakobs (fjakobs)
+
```

Comment:
hmmm with existing files, I am not so sure ... there should be some input from @ecker on this ... he is working with 1&1 legal ... but for new files ... please use MIT.